File: frontend/src/api/models/Briefing.ts
import { UserDto } from './User';
import { BaseDtoDictionary } from './BaseModel';
import { FileDescriptionDto } from './FileDescription';
import { BriefingTypeDto } from './BriefingType';

export interface BriefingDto extends BaseDtoDictionary {
    description?: string;

    created: Date;

    instructor: UserDto;

    mediaUrl?: string;

    textFile: FileDescriptionDto;

    type: BriefingTypeDto;
}

export interface BriefingCreateUpdateDto extends BaseDtoDictionary  {
    description?: string;

    created: Date;

    instructorId: string;

    mediaUrl?: string;

    textFile?: File;

    typeId: string;
}

File: backend/SafetyAndHealth.Db/Models/Briefing.cs
using System;
namespace SafetyAndHealth.Db.Models
{
    public class Briefing : RelationalDictionary
    {
        public string Description { get; set; }

        public DateTime Created { get; set; }

        public User Instructor { get; set; }

        public string InstructorId { get; set; }

        public string MediaUrl { get; set; }

        public FileDescription TextFile { get; set; }

        public long TextFileId { get; set; }

        public BriefingType Type { get; set; }

        public long TypeId { get; set; }
    }
}
